frontend/src/utils/action.tsx

```diff
@@ -51,6 +67,10 @@ export const RemoveFromRateLater = (asyncCallback?: () => void) => {
           onClick={async () => {
             await UsersService.usersMeVideoRateLaterDestroy(video_id);
             if (asyncCallback) await asyncCallback();
+            showSuccessAlert(
+              enqueueSnackbar,
+              'The video has been deleted from your rate later list.'
+            );
```

Comment:
Do we never expect the Destroy to fail ? If it can fail, then a similar logic as for the Create would be best.

Reply:
The only solution I see is that the user has two parallel pages and he decides two press the button delete on both pages.
I implemented the logic got add button because the button appears even if it is already in the rate later list